ECommerceAPI/apps/department/department_url.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path, include
+from ECommerceAPI.apps.department import views
+
+urlpatterns = [
+    path("department/",views.DepartmentView.as_view()),
+    path("departments/<int:id>/",views.DepartmentDetailView.as_view())
```

Comment:
`departments` as plural or `department` as singular.

Reply:
making the correction now